README.md

```diff
@@ -23,16 +23,17 @@ First-of-its-kind patent-pending technology that automatically correlates outbou
 2. In the workflow logs, you will see a link to security insights and recommendations.  
 
     <p align="left">
-      <img src="https://step-security-images.s3.us-west-2.amazonaws.com/build_log_link1.png" alt="Link in build log" >
+      <img src="https://github.com/step-security/supply-chain-goat/blob/main/images/harden-runner/ActionLog.png" alt="Link in build log" >
```

Comment:
@arjundashrath I would like this image to not be too long. It should basically just show one or two actions below harden-runner, but does not have to show all the remaining actions. Also, the green link should be prominent and clearly visible. You can try to highlight it. Also, see if you can take screenshot from the `example link` run